Sources/CircuitBreaker/Invocation.swift

```diff
@@ -70,21 +73,37 @@ public class Invocation<A, B> {
     return false
   }
 
-  /// Marks invocation as having timed out.
+  /// Marks invocation as having timed out if and only if the execution
+  /// of the invocation has not completed yet. In such case, true is returned;
+  /// otherwise, false.
+  /// ### Usage Example: ###
+  /// ```swift
+  /// result = invocation.notifyTimedOut()
+  /// ```
+  public func notifyTimedOut() -> Bool {
+    let boolResult = nofityTimedOut()
```

Comment:
This could just be `return nofityTimedOut()`

Reply:
Good point - thanks I've changed this as suggested.